tests/data/app/Queries/DummyQuery.php

```diff
@@ -9,13 +9,13 @@
 class DummyQuery extends Query
 {
 
-    public function __construct()
+    public function __construct(?string $model = null)
```

Comment:
passiert da mit dem model was?

Reply:
Das ist ein älterer Change aus einem früheren PR - das hat nur im Test gefehlt. Da fehlt noch mehr in den Tests... 

Queries unterscheiden sich model-bezogene und model-unabhängige queries. Erstere kriegen den namen des Models in den constructor übergebene, letztere kriegen null.

alles was `extends Query` macht und den constructor overrided, muss natürlich die signatur matchen.